### PR TITLE
Feature/fsar 62 404 403 pages

### PR DESCRIPTION
**Pull request by EmilyHazlehurst**
Please provide a thorough description of your component and its usage with screenshots.

## Checklist before opening a PR

Before opening a PR, please make sure:

- [ ] your work is in a branch based on and synced with the develop branch.
- [ ] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [ ] your newComponent is imported in the index.js file.
- [ ] your folder hierarchy follows the proper structure.
- [ ] your code follows the coding standards and naming conventions defined.
- [ ] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [ ] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [ ] your code has been tested in all the different screensizes defined.

### CODE REVIEW

File: src/components/general/ErrorResponse/errorResponse.stories.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+export const Error403WithArchive = Template.bind({});
+Error403WithArchive.args = {
+  error: '403',
+  content: '<h1>Access denied</h1>\n' +
+    '<p>The page you were looking for has been moved or does not exist.</p>\n' +
+    nationalArchiveLink({
+      link: '#',
+      link_label: 'This page has been archived by The National Archives and is available through the UK Government Web Archive. (Opens in a new window)',
```
Comment (c-ancia) [inline src/components/general/ErrorResponse/errorResponse.stories.js:28]: is the (Opens in a new window) visible? Have a feeling it should be hidden

File: src/lib.scss [hunk 2]
```diff
@@ -575,7 +575,7 @@ $mobileGutter: 1rem;
   ol {
     list-style: roman;
   }
-  p, h1, h2, h3, h4, h5, h6, ul, ol {
+  > p, h1, h2, h3, h4, h5, h6, ul, ol {
```
Comment (c-ancia) [inline src/lib.scss:578]: I think you need to add the > in front of each element unfortunately

File: src/pages/error/error403.html.twig [hunk 3]
```diff
@@ -0,0 +1,1115 @@
...
+      <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+          <g transform="translate(-1440.000000, -44.000000)" fill="#FFFFFF">
+              <g id="fsa-youtube" transform="translate(1440.000000, 44.000000)">
+                  <path d="M0,22 C0,9.9 9.9,0 22,0 C34.1,0 44,9.9 44,22 C44,34.1 34.1,44 22,44 C9.9,44 0,34.1 0,22 Z M33.55,17.6 C33.55,17.6 33.33,15.95 32.67,15.18 C31.79,14.19 30.8,14.19 30.36,14.19 C27.06,13.97 22.11,13.97 22.11,13.97 L22.11,13.97 C22.11,13.97 17.16,13.97 13.86,14.19 C13.42,14.3 12.43,14.3 11.55,15.18 C10.78,15.95 10.45,17.6 10.45,17.6 C10.45,17.6 10.23,19.58 10.23,21.45 L10.23,23.32 C10.23,25.3 10.45,27.17 10.45,27.17 C10.45,27.17 10.67,28.82 11.33,29.59 C12.21,30.58 13.42,30.47 13.97,30.58 C15.95,30.69 22,30.8 22,30.8 C22,30.8 26.95,30.8 30.25,30.58 C30.69,30.58 31.68,30.47 32.56,29.59 C33.22,28.82 33.44,27.17 33.44,27.17 C33.44,27.17 33.66,25.19 33.66,23.32 L33.66,21.45 C33.77,19.47 33.55,17.6 33.55,17.6 Z"></path>
+                  <polygon id="Path" points="19.8 25.85 19.8 19.14 26.18 22.55"></polygon>
+              </g>
+          </g>
+      </g>
+  </svg>',
+} %}
```
Comment (c-ancia) [inline src/pages/error/error403.html.twig:163]: we have a bunch of svg variables in svg-icons.js. Can you check if you're reusing some of these or if perhaps you can move your variables in this file?

File: src/pages/error/error403WithArchive.html.twig [hunk 4]
```diff
@@ -0,0 +1,1121 @@
...
+      <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+          <g transform="translate(-1440.000000, -44.000000)" fill="#FFFFFF">
+              <g id="fsa-youtube" transform="translate(1440.000000, 44.000000)">
+                  <path d="M0,22 C0,9.9 9.9,0 22,0 C34.1,0 44,9.9 44,22 C44,34.1 34.1,44 22,44 C9.9,44 0,34.1 0,22 Z M33.55,17.6 C33.55,17.6 33.33,15.95 32.67,15.18 C31.79,14.19 30.8,14.19 30.36,14.19 C27.06,13.97 22.11,13.97 22.11,13.97 L22.11,13.97 C22.11,13.97 17.16,13.97 13.86,14.19 C13.42,14.3 12.43,14.3 11.55,15.18 C10.78,15.95 10.45,17.6 10.45,17.6 C10.45,17.6 10.23,19.58 10.23,21.45 L10.23,23.32 C10.23,25.3 10.45,27.17 10.45,27.17 C10.45,27.17 10.67,28.82 11.33,29.59 C12.21,30.58 13.42,30.47 13.97,30.58 C15.95,30.69 22,30.8 22,30.8 C22,30.8 26.95,30.8 30.25,30.58 C30.69,30.58 31.68,30.47 32.56,29.59 C33.22,28.82 33.44,27.17 33.44,27.17 C33.44,27.17 33.66,25.19 33.66,23.32 L33.66,21.45 C33.77,19.47 33.55,17.6 33.55,17.6 Z"></path>
+                  <polygon id="Path" points="19.8 25.85 19.8 19.14 26.18 22.55"></polygon>
+              </g>
+          </g>
+      </g>
+  </svg>',
+} %}
```
Comment (c-ancia) [inline src/pages/error/error403WithArchive.html.twig:162]: same question here with the svg-icons.js file please.

File: src/pages/error/error404.html.twig [hunk 5]
```diff
@@ -0,0 +1,1116 @@
...
+      <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+          <g transform="translate(-1440.000000, -44.000000)" fill="#FFFFFF">
+              <g id="fsa-youtube" transform="translate(1440.000000, 44.000000)">
+                  <path d="M0,22 C0,9.9 9.9,0 22,0 C34.1,0 44,9.9 44,22 C44,34.1 34.1,44 22,44 C9.9,44 0,34.1 0,22 Z M33.55,17.6 C33.55,17.6 33.33,15.95 32.67,15.18 C31.79,14.19 30.8,14.19 30.36,14.19 C27.06,13.97 22.11,13.97 22.11,13.97 L22.11,13.97 C22.11,13.97 17.16,13.97 13.86,14.19 C13.42,14.3 12.43,14.3 11.55,15.18 C10.78,15.95 10.45,17.6 10.45,17.6 C10.45,17.6 10.23,19.58 10.23,21.45 L10.23,23.32 C10.23,25.3 10.45,27.17 10.45,27.17 C10.45,27.17 10.67,28.82 11.33,29.59 C12.21,30.58 13.42,30.47 13.97,30.58 C15.95,30.69 22,30.8 22,30.8 C22,30.8 26.95,30.8 30.25,30.58 C30.69,30.58 31.68,30.47 32.56,29.59 C33.22,28.82 33.44,27.17 33.44,27.17 C33.44,27.17 33.66,25.19 33.66,23.32 L33.66,21.45 C33.77,19.47 33.55,17.6 33.55,17.6 Z"></path>
+                  <polygon id="Path" points="19.8 25.85 19.8 19.14 26.18 22.55"></polygon>
+              </g>
+          </g>
+      </g>
+  </svg>',
+} %}
```
Comment (c-ancia) [inline src/pages/error/error404.html.twig:162]: and also here please

**Review by c-ancia — COMMENTED** [review]
I also feel like the variables you use in the different error pages could be centralised somehow?

File: src/components/general/ErrorResponse/errorResponse.stories.js (hunk 1 above)
Comment (EmilyHazlehurst) [reply src/components/general/ErrorResponse/errorResponse.stories.js:28]: It's visible on the live site so I've left it as is.

File: src/lib.scss (hunk 2 above)
Comment (EmilyHazlehurst) [reply src/lib.scss]: Fixed

File: src/pages/error/error403.html.twig (hunk 3 above)
Comment (EmilyHazlehurst) [reply src/pages/error/error403.html.twig]: I've removed these and added them to the stories file, imported from svg-icons.js.

File: src/pages/error/error403WithArchive.html.twig (hunk 4 above)
Comment (EmilyHazlehurst) [reply src/pages/error/error403WithArchive.html.twig]: I've removed these and added them to the stories file, imported from svg-icons.js.

**Review by c-ancia — APPROVED** [review]

**Pull request merged**